src/simulation/m_start_up.fpp

```diff
@@ -1127,11 +1127,11 @@ contains
             if (num_procs == 1) then
                 time_final = time_avg
                 io_time_final = io_time_avg
-                print *, "Final Time", time_final
+                print *, "Final Time", time_final*1000000000, " ns/gp/rhs"
             else
                 time_final = maxval(proc_time)
                 io_time_final = maxval(io_proc_time)
-                print *, "Final Time", time_final
+                print *, "Final Time", time_final*1000000000, " ns/gp/rhs"
```

Comment:
it looks like this print statement doesn't need to go in the `if` or `else` condition and can just go after

Reply:
agree